### PR TITLE
D8-207

### PR DESCRIPTION
**Pull request by Jov876**
Grants the ability to upload images to the server. The component may be added wherever it is necessary. This includes placing it in admin only areas wherever needed.

### CODE REVIEW

File: pages/upload-button-testing.js
Comment (Finesseology) [inline pages/upload-button-testing.js:1]: Need this to not be a test file, need this to be implemented on the admin dashboard under the "Upload image" menu selection.

File: pages/api/upload.js
Comment (Finesseology) [inline pages/api/upload.js:1]: api folder is public, ensure only the client can upload images from the admin page.

**Review by Finesseology — CHANGES_REQUESTED** [review]
Please make the small tweaks and we can merge

File: pages/upload-button-testing.js
Comment (Jov876) [reply pages/upload-button-testing.js:1]: Testing file has been removed and has been implemented under the upload Image menu selection

File: pages/api/upload.js
Comment (Jov876) [reply pages/api/upload.js:1]: Added code that checks user status for admin credentials

**Review by Finesseology — APPROVED** [review]
LGTM

**Pull request merged**